views/templates/admin/admin_order.tpl

```diff
@@ -16,10 +16,12 @@
 * @license   http://opensource.org/licenses/AFL-3.0 Academic Free License ("AFL"), in the version 3.0
 *}
 
+<link rel="stylesheet" type="text/css" href="{$mod_dir|escape:html:'UTF-8'}views/css/admin-order.css">
```

Comment:
This CSS should be included by the hook hookBackOfficeHeader, after checking that you are on the properly page. Please use the functions addCss and addJs, as made on the Google Analytics module: https://github.com/PrestaShop/ganalytics/blob/master/ganalytics.php#L609-L625